pywebdriver/plugins/scale_driver.py

```diff
@@ -21,6 +21,13 @@
 
 driver = None
 
+weight_converter_table = {
```

Comment:
a last request: as this is a “constant”, it should be in capitals, and here is a more meaningful name, i think.

```suggestion
WEIGHT_UNIT_TO_KG = {
```